### PR TITLE
[gui-tests][full-ci] lint helper files

### PR DESCRIPTION
**Pull request by saw-jan**
Lint helper python files

### CODE REVIEW

File: test/gui/shared/scripts/helpers/FilesHelper.py [hunk 1]
```diff
@@ -27,36 +30,36 @@ def prefix_path_namespace(path):
...
+    return write and os.access(resource, os.W_OK)
 
 
 def read_file_content(file):
-    f = open(file, "r")
-    content = f.read()
-    f.close()
+    with open(file, "r", encoding="utf-8") as f:
+        content = f.read()
+        f.close()
```
Comment (amrita-shrestha) [inline test/gui/shared/scripts/helpers/FilesHelper.py:62]:
```suggestion
```
i think `with` close file automatically 😄

Comment (saw-jan) [reply test/gui/shared/scripts/helpers/FilesHelper.py:62]: yes, true

File: test/gui/shared/scripts/helpers/ConfigHelper.py [hunk 2]
```diff
@@ -158,22 +155,20 @@ def init_config():
...
     return CONFIG
 
 
 def set_config(key, value):
-    global CONFIG, READONLY_CONFIG
     if key in READONLY_CONFIG:
-        raise Exception('Cannot set read-only config: %s' % key)
+        raise KeyError(f'Cannot set read-only config: {key}')
     CONFIG[key] = value
     return CONFIG
```
[the remark this answers is not in the document]
Comment (saw-jan) [reply test/gui/shared/scripts/helpers/ConfigHelper.py]: really don't have to. :+1:

**Review by PrajwolAmatya — APPROVED** [review]
LGTM

**Pull request merged**